### PR TITLE
25. Make app more responsive for mobile compatibility

### LINKED ISSUE #55: 25. Make app more responsive for mobile compatibility
## Summary

The app should be mobile friendly. Using tailwind, make sure things are responsive.

Things to include: nav bar, font sizes, app padding, button sizes.

## Acceptance Criteria

- [x] App looks good for mobile sizing
- [x] Slide in/out Nav bar

### PR DESCRIPTION
**Pull request by eonflower**
## Description

- Updated the sizing of components based on screen size
- Updated Logo
- Updated Login page styling

## Related Issue

closes #55 

## Acceptance Criteria

- [x] App looks good for mobile sizing
- [x] Slide in/out Nav bar

## Type of Changes

enhancement, accessibility

### Before
Screenshot 2024-04-02 at 10 06 56 PM

https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/34324c6b-95db-4814-96ee-2edbf9223027

### After

Screenshot 2024-04-02 at 10 06 42 PM

https://github.com/the-collab-lab/tcl-66-smart-shopping-list/assets/98231995/340eec9c-ff96-4c8b-9fd0-5b10f1c3482b

## Testing Steps / QA Criteria

1. git pull from main
2. git checkout an-issue-25
3. inspect page and check responsiveness of different devices

### CODE REVIEW

File: src/views/List.jsx [hunk 1]
```diff
@@ -61,7 +62,12 @@ export function List({ data, listPath, lists, loading }) {
 				<Spinner />
 			) : data.length > 0 ? (
 				<>
-					<h2 className="flex justify-center xsm:text-md sm:text-lg md:text-3xl mt-16 mb-12">
+					<img
+						src={pageTitle}
+						alt="The Collab Lab"
+						className="mx-auto xsm:h-[24px] w-auto sm:hidden"
```
Comment (3campos) [inline src/views/List.jsx]: In the list view, the logo covers the share list button and the avatar count in the iPhoneSE view. I have some proposed code below that corrects this problem which you can copy and paste if you want.

## Current iPhoneSE view
Screen Shot 2024-04-03 at 7 24 50 PM

## Using My Below Code
Screen Shot 2024-04-03 at 7 18 29 PM

className="flex justify-start ml-6 xsm:h-[24px] w-auto sm:hidden"

**Review by 3campos — CHANGES_REQUESTED** [review]
@eonflower, Awesome job on the logo and the responsiveness. I love how the app looks, especially in mobile view! Great job on making the code in the Navigation Bar component more DRY.

There's one change to make for the List view and then you can merge. I left a comment about it containing screenshots and included a potential solution that you can copy and paste if you wish.

**Comment by eonflower** [comment]
Got it! I fixed it @3campos :)

**Comment by 3campos** [comment]
@eonflower Just to confirm what I'm seeing, the logo has been removed from the top of the list view. If that was intentional, then you're good to merge.

**Review by 3campos — APPROVED** [review]

**Comment by eonflower** [comment]
@3campos yeah I removed it from the main page since it's in the nav bar. I felt like it wasn't necessary to have it in both spots.

**Pull request merged**